Add missing short description in HtmlEditor styling mode article (#2169)
DevExpress/devextreme-documentation: api-reference/10 UI Components/dxHtmlEditor/1 Configuration/stylingMode.md

---
---
##### shortDescription

> Specifies how HtmlEditor's toolbar and content field are styled in your application.

Reviewer: ```suggestion
Specifies how the HtmlEditor's [toolbar](/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/toolbar/) and content field are styled.
```
